### PR TITLE
Handle `InterfaceError`

### PR DESCRIPTION
**Pull request by tayloraburgess**

### CODE REVIEW

**Comment by tayloraburgess** [comment]
[ch6237]

File: mbq/ranch/lib/error_handling.py [hunk 1]
```diff
@@ -12,6 +15,13 @@ def log_errors_and_send_to_rollbar(func):
     def wrapper(*args, **kwargs):
         try:
             func(*args, **kwargs)
+        except psycopg2.InterfaceError:
```
Comment (mattbriancon) [inline mbq/ranch/lib/error_handling.py]: This won't work in os-core cuz it uses mysql.

Django should be wrapping the exception in a db-agnostic way. We should catch that instead.

Comment (tayloraburgess) [reply mbq/ranch/lib/error_handling.py]: 🤦‍♀️ Right. I was staring at the django wrapper exception in papertrail the whole time I was writing this PR, and still used psycopg instead

**Review by mattbriancon — APPROVED** [review]
I can't tell if your comment is out of date (SIGTERM) or intentional. You decide then :ship:

**Pull request merged**